WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -963,6 +963,7 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
                 let index = strongSelf.filterSettings.availablePostListFilters().indexOf(selectedValue as! PostListFilter) {
 
                 strongSelf.filterSettings.setCurrentFilterIndex(index)
+                strongSelf.syncItemsWithUserInteraction(false)
```

Comment:
I think it might best to call `syncItemsWithUserInteraction` after `refreshAndReload` as we would have already appropriately set up the fetch request, table, etc to be ready for any newly synced content.